### ISSUE TITLE
Replace Jupyter Notebooks with Python Scripts

### ISSUE THREAD

**Issue by J-Yash** [cns-iu/hra-amap#4]
For ease of use in production settings as well as to dockerize the application, it might be better to replace the Jupyter notebook workflow with a python script workflow.

**Comment by desaibhargav** [comment]
Hi Yash, the Jupyter Notebook workflow is just meant as a quick start on how to use the repository and some markdown cells for explanations, it goes well with a publication. But you make a good point. I will add a CLI interface to the code, it'll play well with Docker as well

**Comment by bherr2** [comment]
Yes, having this work as a standard python library / scripts is an absolute requirement for moving to production. For the CLI I recommend Python's [argparse module](https://docs.python.org/3/library/argparse.html).

**Comment by bherr2** [comment]
Duplicate of #7
